pkg/certs/cert-inspection/certgraphanalysis/collector.go

```diff
@@ -159,14 +159,12 @@ func PKIListFromParts(ctx context.Context, inClusterResourceData *certgraphapi.P
 	for i := range certs {
 		certList.Items = append(certList.Items, *certs[i])
 	}
-	guessLogicalNamesForCertKeyPairList(certList, nodes)
```

Comment:
You will owe something functional for the cluster-debug-tools document generation in 4.15 once we make this change.  You ready?

Reply:
Fairly certain all teams can complete descriptions by 4.15 GA